src/java.base/aix/classes/jdk/internal/loader/ClassLoaderHelper.java

```diff
@@ -39,18 +39,20 @@ class ClassLoaderHelper {
     private ClassLoaderHelper() {}
 
     /**
-     * Returns true if loading a native library only if
-     * it's present on the file system.
+     * Shared objects may be loaded from .a archive object on AIX and .so may not exist.
+     * This method returns false so that loading of shared library continues if
+     * libname.so is not present.
      */
     static boolean loadLibraryOnlyIfPresent() {
-        return true;
+        return false;
     }
 
     /**
-     * Returns an alternate path name for the given file
-     * such that if the original pathname did not exist, then the
-     * file may be located at the alternate location.
-     * For AIX, this replaces the final .so suffix with .a
+     * AIX implementation of JVM_LoadLibrary handles the alternate path name mapping.
+     * If loading of the given library name with ".so" suffix fails, it will attempt
+     * to load the library of the same name with ".a" suffix as the alternate name.
+     * This method simply returns null.  It could implement the alternate name
+     * converting ".so" with ".a" suffix but redundant.
```

Comment:
Please don't change this comment; the one from upstream is incorrect for OpenJ9.

Reply:
Restore the comment.